lib/pliny/tasks/db.rake

```diff
@@ -34,7 +34,7 @@ begin
     task :seed do
       if File.exist?('./db/seeds.rb')
         database_urls.each do |database_url|
-          Sequel.connect(database_url)
+          DB = Sequel.connect(database_url)
```

Comment:
But if there's multiple URLs, the `DB` might be pointing to an arbitrary database (i.e. it's order dependent). Does that really mean it does the correct thing? I don't know enough about the scenarios in which there are multiple URLs to provide a recommendation. If you know for sure that it's always the first one, then making `each` a `map` that returns the connection and saving the first one would be better, for example.